tests/test_oc4ids_tranforms.py

```diff
@@ -0,0 +1,1332 @@
+                            "uri": "http://www.geonames.org/2640729/oxford.html",
+                        },
+                        "deliveryAddress": {
+                            "postalCode": "OX1 1BX",
+                            "countryName": "United Kingdom",
+                            "streetAddress": "Town Hall, St Aldate's",
+                            "region": "Oxfordshire",
+                            "locality": "Oxford",
+                        },
+                    }
+                ],
+            },
+        }
+    ]
+
+    output = transforms._run_transforms(copy.deepcopy(releases), "1", transforms=[transforms.location],)
+
+    assert "locations" not in output
+
+
+def test_budget():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "planning": {"budget": {"amount": {"amount": "1000", "currency": "USD"}}},
+        }
+    ]
+
+    output = transforms._run_transforms(copy.deepcopy(releases), "1", transforms=[transforms.budget],)
+    assert output["budget"]["amount"] == releases[0]["planning"]["budget"]["amount"]
+
+
+def test_budget_multiple():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "planning": {"budget": {"amount": {"amount": "1000", "currency": "USD"}}},
+        },
+        {
+            "ocid": "ocds-213czf-2",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T06:07:08Z",
+            "planning": {"budget": {"amount": {"amount": "1234", "currency": "USD"}}},
+        },
+    ]
+
+    output = transforms._run_transforms(copy.deepcopy(releases), "1", transforms=[transforms.budget],)
+    total = float(releases[0]["planning"]["budget"]["amount"]["amount"]) + float(
+        releases[1]["planning"]["budget"]["amount"]["amount"]
+    )
+    assert output["budget"]["amount"]["amount"] == total
+    assert output["budget"]["amount"]["currency"] == releases[0]["planning"]["budget"]["amount"]["currency"]
+
+
+def test_budget_fail():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "planning": {"budget": {"amount": {"amount": "999", "currency": "USD"}}},
+        },
+        {
+            "ocid": "ocds-213czf-2",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T06:07:08Z",
+            "planning": {"budget": {"amount": {"amount": "6464", "currency": "EUR"}}},
+        },
+    ]
+
+    output = transforms._run_transforms(copy.deepcopy(releases), "1", transforms=[transforms.budget],)
+    # Different currencies could not be totalled
+    assert "budget" not in output
+
+
+def test_budget_approval():
```

Comment:
Should this test also cover an input with multiple contracting processes, each with a 'budgetApproval' in `planning/documents`?